### PR TITLE
Update for finalfusion-python 0.7

### PR DESCRIPTION
**Pull request by danieldk**

### CODE REVIEW

**Comment by danieldk** [comment]
I'll probably add CI soon-ish as well.

File: finalfusion_inspector/metadata.py [hunk 1]
```diff
@@ -94,11 +93,11 @@ def switchEmbeddings(self, embeddings):
     def updateMetadata(self):
         self._metadata = []
 
-        if not isinstance(self.embeddings.metadata, str):
+        if not isinstance(self.embeddings.metadata, dict):
```
Comment (sebpuetz) [inline finalfusion_inspector/metadata.py:96]: `Metadata` subclasses `dict`, but since it's Python, someone could throw other types in there, to...

**Review by sebpuetz — APPROVED** [review]
Looks good

File: finalfusion_inspector/metadata.py (hunk 1 above)
Comment (danieldk) [reply finalfusion_inspector/metadata.py:96]: True, then they won't see metadata I guess? Good punishment ;).

**Pull request merged**